### ISSUE TITLE
Pretraining model './data_smc/cora_model_128.pt' disappeared. Can you provide it?

### ISSUE THREAD

**Issue by octopusStar218** [MingChen-Sun/GPPT#3]
Pretraining model './data_smc/cora_model_128.pt' disappeared. Can you provide it?

**Comment by OPilgrim** [comment]
same problem

**Comment by zzyzeyuan** [comment]
Looking forward your reply.

**Comment by baibaibaialone** [comment]
same problem +1

**Comment by SONGLEI-arch** [comment]
same problem

**Comment by hcmdgh** [comment]
same problem

**Comment by Opao-heng** [comment]
请问解决了吗

**Comment by MM-YY-WW** [comment]
I generate the pre-trained models by running the train_sampling_unsupervised.py file. I'm not certain if this reproduction is correct because adding or removing the pre-trained model doesn't appear to influence the results. Moreover, increasing the cluster number adversely affects accuracy. Cluster number equals to 1 performs the best.
